Bio/Phylo/_utils.py

```diff
@@ -473,8 +476,17 @@ def draw_clade(clade, x_start, color, lw):
     else:
         if name:
             axes.set_title(name)
-    axes.set_xlabel("branch length")
-    axes.set_ylabel("taxa")
+        else:
+            if title:
```

Comment:
You could use a single "elif" line here.

Reply:
Got it.